src/isekai.cr

```diff
@@ -0,0 +1,67 @@
+require "option_parser"
+require "./parser.cr"
+
+module Isekai
+    # Structure holding the options passed by the user
+    # on the command line
+    struct ProgramOptions
+        # Additional arguments to pass to libclang (e.g. -I)
+        property clang_args = ""
+        # Dynamic loop unrolling limit
+        property loop_sanity_limit = 0
+        # Print progress during the execution
+        property progress = false
+        # Arithmetic circuit output file
+        property arith_file = ""
+        # Boolean circuit output file
+        property bool_file = ""
+        # Number of bits in the word
+        property bit_width = 32
+        # Print the resulting expressions
+        property print_exprs = false
+        # Ignore overflow
+        property ignore_overflow = false
+    end
+
+    class ParserProgram
+        # Main 
+        def main
+            opts = ProgramOptions.new
+
+            OptionParser.parse! do |parser|
+                parser.banner = "Usage: isekai [arguments] file"
+                parser.on("-c", "--cpparg=ARGS", "Extra arguments to clang")\
+                    { |args| opts.clang_args = args }
+                parser.on("-a", "--arith=FILE", "Arithmetic circuit output file")\
+                    { |file| opts.arith_file = file }
+                parser.on("-b", "--bool=FILE", "Boolean circuit output file")\
+                    { |file| opts.bool_file = file }
+                parser.on("-w", "--bit-width", "Bit width")\
```

Comment:
Can you explain what is the bit width option? In the documentation maybe.

Reply:
I've added a bit more extended description in the `struct ProgramOptions` and I put a comment here to refer to that struct for more detailed descriptions. I also updated this brief description.